tests/test_sqlite_herd.py

```diff
@@ -10,11 +10,11 @@
             'CLIENT_CLASS': 'django_redis.client.HerdClient',
         }
     },
-    'doesnotexist': {
-        'BACKEND': 'django_redis.cache.RedisCache',
-        'LOCATION': '127.0.0.1:56379:1',
-        'OPTIONS': {
-            'CLIENT_CLASS': 'django_redis.client.HerdClient',
+    "doesnotexist": {
+        "BACKEND": "django_redis.cache.RedisCache",
```

Comment:
Why are these files changing? It isn't clear why that is necessary for the tests to pass.

Reply:
Oh I see. It was changed to use a bogus port. Is that right? :+1: